src/config/dynamic-theme-fixes.config

```diff
@@ -4298,8 +4305,6 @@ INVERT
 .svg-Wikimedia-logo_black
 .header .branding-box > a > span > img
 .main-footer-menuToggle
-div.post-content.footer-content > h2 > img
```

Comment:
Nice, but we don't want to loose changes already included - can you fix this in your PR?

Reply:
I messed up, idk when did this happened.